rest-demo/src/main/java/com/projectexample/restdemo/controller/CloudVendorController.java

```diff
@@ -29,34 +29,34 @@ public ResponseEntity<Object> getCloudVendorDetails( @Valid @PathVariable("Vendo
 
     }
     @GetMapping
-    public ResponseEntity <List<CloudVendor>> getAllCloudVendorDetails()
+    public ResponseEntity <List<CloudVendor>> getAllCloudendorDetails()
     {
         List<CloudVendor>list=cloudVendorService.getAllCloudVendor();
-        if(list.size()<=0)
+        if(list.isEmpty())
         {
             return  ResponseEntity.status(HttpStatus.NOT_FOUND).build();
         }
         return  ResponseEntity.of(Optional.of(list));
-       // return ResponseHandler.responseBuilder("All Vendor Details are given here",HttpStatus.OK,cloudVendorService.getAllCloudVendor());
+
 
     }
     @PostMapping
     public  ResponseEntity<Object> CreateCloudVendorDetails(@RequestBody CloudVendor cloudVendor)
     {
-       return ResponseHandler.responseBuilder("cloud vendor created",HttpStatus.CREATED,cloudVendorService.createCloudVendor( cloudVendor));
+       return ResponseHandler.responseBuilder("The Vendor was created successfully",HttpStatus.CREATED,cloudVendorService.createCloudVendor( cloudVendor));
 
 
 
     }
     @PutMapping
     public ResponseEntity<Object> updateCloudVendorDetails(@RequestBody CloudVendor cloudVendor)
     {
-        return ResponseHandler.responseBuilder("cloud vendor updated",HttpStatus.OK,cloudVendorService.updateCloudVendor( cloudVendor));
+        return ResponseHandler.responseBuilder("The Vendor was update successfully",HttpStatus.OK,cloudVendorService.updateCloudVendor( cloudVendor));
```

Comment:
return type should be ResponseEntity<loudeVendor> not ResponseEntity<Object>
Object is very generic